src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx

```diff
@@ -85,17 +86,18 @@ Keep in mind that the default timeout for a new monitor is 180 seconds.
   >
     1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select **Synthetics**. Click <Icon name="fe-plus-circle"/> [**Create monitor**](https://one.newrelic.com/synthetics-nerdlets/monitor-create) in the far right of the Explorer nav.
     2. Specify a [monitor type](#setting-type) and [name](#setting-name).
-    3. Select the [locations](#setting-location) from which you want your monitor to run.
-    4. Choose a [frequency](#setting-frequency) to determine how often each location will run your monitor.
-    5. Optional: Set up [alert notifications](/docs/synthetics/new-relic-synthetics/using-monitors/alerting-synthetics).
-    6. Select **Next: Write your script** to create a script for your [**scripted browser**](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) or [**API test**](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests), then select **Validate** to verify your syntax.
+    3. Choose a [period](#setting-frequency) to determine how often each location will run your monitor.
+    4. Optional: Configure [device emulation](/docs/synthetics/new-relic-synthetics/using-monitors/device-emulation) settings, including device type and screen orientation are available for scripted browser monitors.
```

Comment:
@bpeckNR this link doesn't seem to work. Should it be this: /docs/synthetics/synthetic-monitoring/using-monitors/device-emulation

Reply:
Great catch! I fixed this link.